pkg/config/config.go

```diff
@@ -48,3 +51,48 @@ func NewKVConfiguration(fileName string) (KVConfiguration, error) {
 	err = decoder.Decode(&configuration)
 	return configuration, err
 }
+
+func (c *KVConfiguration) GetReplications() []string {
+	var localStores []string
+	var neighborStores []string
+	var remoteStores []string
+	for _, store := range c.Stores {
+		switch region := store.RegionType; region {
+		case "local":
+			localStores = append(localStores, fmt.Sprintf("%s:%d", store.Host, store.Port))
+		case "neighbor":
+			neighborStores = append(neighborStores, fmt.Sprintf("%s:%d", store.Host, store.Port))
+		case "remote":
+			remoteStores = append(remoteStores, fmt.Sprintf("%s:%d", store.Host, store.Port))
+		}
+	}
+	n := len(localStores)
```

Comment:
not very clear what the purpose of this block is. could this be revised to make the intent more clear?

Reply:
Please refer to the design from Du Peng @ https://user-images.githubusercontent.com/252020/184443299-799f1ed4-493a-4ea2-a9ed-72e15a2737af.png

Since you might miss some design proposals, I will add it to the codes for better references.